workers/caching/metaphysics-cdn.ts

```diff
@@ -77,6 +77,7 @@ export default {
             "Access-Control-Allow-Headers": allowedHeaders,
             "Access-Control-Allow-Methods": "GET,POST,OPTIONS",
             "Access-Control-Allow-Origin": "*",
+            "Access-Control-Expose-Headers": "Cf-Cache-Status",
```

Comment:
My understanding was that this header was required on the response itself, not the preflight response.

Reply:
Wasn't 100% clear to me (I assumed the preflight) - will switch